MedicalDocManagment.WebUI/app/main/children-cards-list/add/children-card-add.component.ts

```diff
@@ -0,0 +1,133 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+// import { NotificationsService, SimpleNotificationsComponent } from 'angular2-notifications';
+import ChildCardModel from '../../models/child-card.model';
+import DiagnosisModel from "../../models/diagnosis.model";
+import DiagnosesModel from "../../models/diagnoses.model";
+import ClassModel from "../../models/class.model";
+import ClassesModel from "../../models/classes.model";
+import BlockModel from '../../models/block.model';
+import BlocksModel from '../../models/blocks.model';
+import NosologyModel from "../../models/nosology.model";
+import NosologiesModel from "../../models/nosologies.model";
+import ChildrensCardService from '../../services/children-card.service';
+import MkhsService from "../../services/mkhs.service";
+
+@Component({
+    moduleId: module.id,
+    selector: 'children-card-add',
+    templateUrl: 'children-card-add.component.html',
+    providers: [ChildrensCardService, MkhsService]
+})
+export default class ChildrenCardAddComponent {
+    childrensCard = new ChildCardModel();
+    limitDate = new Date();
+    isDatesErrorsVisible = false;
+    isCheckInErrorsVisible = false;
+    isCheckOutErrorsVisible = false;
+    selectedClass = new ClassModel();
+    classes: ClassesModel;
+    selectedBlock = new BlockModel();
+    blocks: BlocksModel;
+    selectedNosology= new NosologyModel();
+    nosologies: NosologiesModel;
+    diagnoses: DiagnosesModel;
```

Comment:
The preferred style for other properties is to initialize the field in the constructor (Google JavaScript Style Guide).